docs/releases/3.2.txt

```diff
@@ -68,6 +68,10 @@ Minor features
 :mod:`django.contrib.gis`
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
+* The GDALRaster.transform() function now accepts the target spatial reference
+  system input in form of integer SRID, spatial reference WKT string,
+  a PROJ.4 string, or a SpatialReference object.
```

Comment:
As far as I'm aware, spatial reference WKT string and a PROJ.4 string were supported without this change (even if it's not documented).

Reply:
Note that it's just "PROJ" now, not "PROJ.4".

Reply:
@rico-ci See #12901 for rename of PROJ.4 to PROJ which was done for Django 3.1 which is the next version to be released. You've linked to the documentation for 3.0.

@felixxm I've also noticed that the link above to the GDAL documentation is raising a 500 error. In fact, it also seems to do so for all versions...

Reply:
@pope1ni Docs should work now.